components/bl-emoji/src/index.js

```diff
@@ -0,0 +1,71 @@
+import { useState, useMemo } from 'react';
+
+import EmojiPicker from './emoji-picker-react.min.js';
+
+const { cn, normalizeDimensionValue } = BackendlessUI.CSSUtils;
+
+export default function EmojiComponent({ component, eventHandlers, elRef }) {
+  const {
+    classList, style, display, disabled, buttonColor, buttonSize,
+    dropdownHeight, dropdownWidth, dropdownPosition, theme, emojiStyle, searchDisabled
+  } = component;
+  const { onEmojiSelect } = eventHandlers;
+
+  const [pickerVisibility, setPickerVisibility] = useState(false);
+
+  const validButtonSize = useMemo(() => normalizeDimensionValue(buttonSize), [buttonSize]);
+  const validDropdownWidth = useMemo(() => normalizeDimensionValue(dropdownWidth), [dropdownWidth]);
+  const validDropdownHeight = useMemo(() => normalizeDimensionValue(dropdownHeight), [dropdownHeight]);
+
+  const handleEmojiButtonClick = () => {
+    setPickerVisibility(prevState => !prevState);
+  };
+
+  const handleEmojiClick = event => {
+    const { emoji, names } = event;
+
+    navigator.clipboard.writeText(emoji);
```

Comment:
But how it will be inserted into text fields? Pasting? It may not be convenient.
We should try to find a way to link it with input and write an emoji directly into it, without a clipboard.